core/adapters/bigquery.ts

```diff
@@ -19,14 +19,19 @@ export class BigQueryAdapter extends Adapter implements IAdapter {
     tableMetadata: dataform.ITableMetadata
   ): Tasks {
     const tasks = Tasks.create();
-    // Drop views/tables first if they exist.
-    if (tableMetadata && tableMetadata.type != this.baseTableType(table.type)) {
+    const shouldWriteIncrementally = this.shouldWriteIncrementally(runConfig, tableMetadata);
```

Comment:
we only use this in one place, let's inline it?

Reply:
👍